server/profileStore.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+require_once 'config.php';
+
+
+
+storeProfile($_POST);
+
+function storeProfile($data) {
+  GLOBAL $DB_CONN;
+  $response = array();
+  $response['success'] = false;
+
+  $DB_CONN->beginTransaction();
+
+  try {
+    $userUpdate = "UPDATE User
+      SET  firstName = ?,lastName = ?,phoneNumber = ?,email = ?,preparesTaxes = ?
+       WHERE userId = ? ;";
+    $userParams = array(
+      $data['firstName'],
+      $data['lastName'],
+      $data['emailNumber'],
+      $data['phoneNumber']
+      $data['preparesTaxes']
+    );
+    $stmt = $DB_CONN->prepare($userUpdate)
+    $stmt->execute($userParams);
+
+    $userabilityId = $DB_CONN->lastInsertId();
+
+//Not exactly sure how to implement language information.. this is what I could understand.
```

Comment:
Let's sit down and talk about this sometime, I'll try to get an example up about this structure